src/runtime/slot-context.ts

```diff
@@ -136,10 +136,15 @@ export class ProvidedSlotContext extends SlotContext {
     consumer.setContent(content, handler);
   }
 
-  get container() { return this._container; }
-  get containerAvailable() { return !!this._container; }
+  get container(): HTMLElement {
+    return this._container;
+  }
+
+  get containerAvailable(): boolean {
+    return !!this._container;
```

Comment:
Tangential (and I know you didn't write it), but personally I don't like `!!` and prefer explicit cast to Boolean.

Reply:
Started a topic in chat about using https://eslint.org/docs/rules/no-implicit-coercion